app/views/posts/index.html.rb

```diff
@@ -0,0 +1 @@
+'This will be index content for posts'
```

Comment:
- [ ] We are missing a template. This is because your template are saved with `.rb` extension instead of `erb`. Kindly correct this.
![Screenshot from 2022-03-23 14-45-28](https://user-images.githubusercontent.com/65860678/159702964-866618fd-3d80-469b-84ee-87f8760fc7a8.png)